vhdbuilder/packer/install-dependencies.sh

```diff
@@ -241,20 +209,8 @@ fi
 ls -ltr /opt/gpu/* >> ${VHD_LOGS_FILEPATH}
 
 installBpftrace
-echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
-
-cat << EOF >> ${VHD_LOGS_FILEPATH}
-  - nvidia-docker2=${NVIDIA_DOCKER_VERSION}
-  - nvidia-container-runtime=${NVIDIA_CONTAINER_RUNTIME_VERSION}
-  - nvidia-gpu-driver-version=${GPU_DV}
-  - nvidia-fabricmanager=${GPU_DV}
-EOF
 
 installBcc
-cat << EOF >> ${VHD_LOGS_FILEPATH}
```

Comment:
similarly, not from OS packages. you could pull the version var out of the install function and log it here, or avoid touching this one :)